apps/condo/domains/organization/access/Organization.js

```diff
@@ -76,10 +76,15 @@ async function canReadOrganizations ({ authentication: { item: user }, listKey }
     return { OR: accessConditions }
 }
 
-async function canManageOrganizations ({ authentication: { item: user }, operation }) {
+async function canManageOrganizations ({ authentication: { item: user }, operation, listKey, originalInput }) {
     if (!user) return throwAuthenticationError()
     if (user.deletedAt) return false
     if (user.isSupport || user.isAdmin) return true
+    const hasDirectAccess = await canDirectlyManageSchemaObjects(user, listKey)
+    if (hasDirectAccess) return true
+
+    // NOTE: The "isApproved" field can only be managed by the admin, support, users with special rights.
+    if ('isApproved' in originalInput) return false
```

Comment:
Should we add a common fields-access function? Seems like we need to protect many fields (like importId, importRemoteSystem and so on)

Reply:
We should block organization creating here. Since the only source of organization should be `registerNewOrganization` mutation

Reply:
> Should we add a common fields-access function? Seems like we need to protect many fields (like importId, importRemoteSystem and so on)

It seems our fields are protected. But not everything is possible